src/armado/cdpindex.py

```diff
@@ -138,7 +138,7 @@ def generate_from_html(dirbase, verbose):
     top_pages = preprocess.pages_selector.top_pages
 
     titles_texts = {}
-    with codecs.open(config.LOG_TITLES, "rt", encoding='utf8') as fh:
+    with codecs.open(config.LOG_TITLES, "r", encoding='utf8') as fh:
```

Comment:
¿Porqué sigue usando codecs? 

```suggestion
    with open(config.LOG_TITLES, "r", encoding='utf8') as fh:
```

Reply:
Se me pasó. Ahi lo cambie.